core/src/main/kotlin/org/evomaster/core/database/SqlInsertBuilder.kt

```diff
@@ -154,6 +160,84 @@ class SqlInsertBuilder(
 
     }
 
+    private class ConstraintCollector : TableConstraintVisitor<List<TableConstraint>, Void> {
+
+        override fun visit(constraint: IsNotNullConstraint?, argument: Void?): List<TableConstraint> {
```

Comment:
why in these visitors are you passing nullable inputs with `?` ? even if extending a Java class, putting them non-nullable should still compile

Reply:
You are right. I removed the ? modifier on all the constraints